src/XenCert/StorageHandler.py

```diff
@@ -3413,15 +3417,19 @@ def Create(self):
 
             device_config = copy.deepcopy(self.device_config)
 
+            if 'targetIQN' in device_config and len(device_config['targetIQN'].split(',')) > 1:
+                device_config['targetIQN'] = '*'
```

Comment:
What is the purpose for setting the `targetIQN` tp `*`?

Reply:
It's copy from LVM handling, it's SM interface.